java/test/src/main/java/test/Ice/stream/Test.ice

```diff
@@ -56,7 +55,7 @@ class OptionalClass
     byte by;
     optional(1) short sh;
     optional(2) int i;
-    optional(3) SmallStruct sm;
+    optional(3) LargeStruct ls;
```

Comment:
We don't have this field in the C++ Ice/stream test.

Reply:
After spending the last few weeks in our tests, I can tell you there's actually lots of these random inconsistencies in the tests. So I'm not surprised. Why did they happen? Who knows.